yt_dlp/extractor/playsuisse.py

```diff
@@ -135,11 +138,29 @@ class PlaySuisseIE(InfoExtractor):
             url
         }'''
 
+    def _perform_login(self, username, password):
+        if username is None or password is None:
+            self.raise_login_required('Your Play Suisse account\'s details must be provided with --username and --password.')
```

Comment:
`_perform_login` won't be called if the user does not provide credentials. This check needs to be done in `_real_extract` or `_real_initialize`. But does passing cookies work (instead of user/pass) work for this site? Ideally we would check if an auth cookie is in the cookiejar (use `self._get_cookies()`) instead of checking for user/pass

Reply:
Thanks for the review!

 Passing cookies doesnt work as the parameter in question is actually saved in local storage, not in a cookie. Is there a way to extract that instead?

Reply:
Local storage cannot be extracted, no